sigh/lib/assets/resign.sh

```diff
@@ -202,7 +202,11 @@ fi
 
 # Set the app name
 # The app name is the only file within the Payload directory
-APP_NAME=$(ls "$TEMP_DIR/Payload/")
+
+#APP_NAME=$(ls "$TEMP_DIR/Payload/")
+
+#fix it!!!.app not the only file within the Payload directory,such as StoreKit folder.
```

Comment:
Improve comment to simply explain what the code is doing / why grep .app is necessary.